docs/design-docs/08-package-variant.md

```diff
@@ -664,28 +1242,53 @@ example, a package should not accept a Porch Repository resource just because
 that happens to be the targeting mechanism. That would make the package unusable
 in other contexts.
 
+#### PackageVariantSet Status
+
+The PackageVariantSet status uses these conditions:
+ - `Stalled` is set to True if there has been a failure that most likely
+   requires user intervention.
+ - `Ready` is set to True if the last reconciliation successfully reconciled
+   all targeted PackageVariant resources.
+
 ## Future Considerations
 - As an alternative to the floating tag proposal, we may instead want to have
   a separate tag tracking controller that can update PV and PVS resources to
   tweak their upstream as the tag moves.
-- Probably want to think about groups of packages (that is, a collection of
-  upstreams with the same set of mutation to be applied). For now, this would be
-  handled with PackageVariant / PackageVariantSet resources that differ only
-  in their upstream / downstream. Theoretically we could do that with label
-  selectors on packages but it gets really ugly really fast. I suspect just
-  making people copy the PV / PVS is better.
+- Installing a collection of packages across a set of clusters, or performing
+  the same mutations to each package in a collection, is only supported by
+  creating multiple PackageVariant / PackageVariantSet resources. Options to
+  consider for these use cases:
+  - `upstreams` listing multiple packages.
+  - Label selector against PackageRevisions. This does not seem that useful, as
+    PackageRevisions are highly re-usable and would likely be composed in many
+    different ways.
+  - A PackageRevisionSet resource that simply contained a list of Upstream
+    structures and could be used as an Upstream. This is functionally equivalent
+    to the `upstreams` option, but that list is reusable across resources.
+  - Listing multiple PackageRevisionSets in the upstream would be nice as well.
+  - Any or all of these could be implemented in PackageVariant,
+    PackageVariantSet, or both.
 
 ## Footnotes
+[^porch17]: Implemented and coming in Porch v0.0.17.
 [^notimplemented]: Proposed here but not yet implemented as of Porch v0.0.16.
 [^setns]: As of this writing, the `set-namespace` function does not have a
     `create` option. This should be added to avoid the user needing to also use
     the `upsert-resource` function. Such common operation should be simple for
-    users. Another option is to build this into PackageVariant, though at this
-    time we do not plan to do so.
-[^pdc]: A prototype version of this was implemented in Nephio PackageDeployment,
-    but this has not been implemented in PackageVariant as of Porch v0.0.16.
-
-## Figure Legend
-
-![Figure Legend](packagevariant-legend.png)
-
+    users.
+[^pvsimpl]: This document describes PackageVariantSet `v1alpha2`, which has not
```

Comment:
@mortent I think we should just go ahead an implement the machinery for conversion webhooks. I am assuming they would run in the controller binary. We're likely to need them in the future. The actual roundtrip conversions in this case are pretty easy to do (the proposed `v1alpha2` API is a superset of the `v1alpha1` functionality).